lib/util/schema.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { z } from 'zod';
 import { logger } from '../../test/util';
 import * as memCache from './cache/memory';
-import { checkSchema, reportErrors } from './schema';
+import * as schema from './schema';
```

Comment:
@zharinov can we prefer non star imports? i don't like those

Reply:
I think it's okay for this particular case: `schema.match(...)` looks most clear comparing to other alternatives to me. What alternative would you like?

Reply:
`matchesSchema(...)` 😉

Reply:
I would like to see something shorter, though 🤔

Reply:
Seems like we'll use it a lot in future, so we need to think about naming

Reply:
yes, 🤔